take.go

```diff
@@ -0,0 +1,107 @@
+package gcf
+
+type takeIterable[T any] struct {
+	itb   Iterable[T]
+	count int
+}
+
+type takeIterator[T any] struct {
+	it      Iterator[T]
+	count   int
+	i       int
+	done    bool
+	current T
+}
+
+// Take makes Iterable with count elements from ahead.
```

Comment:
```suggestion
// Take makes Iterable with counted elements from ahead.
```
target: Take
こちらの方が訳として良いものになった